scripts/defutil.py

```diff
@@ -26,12 +30,24 @@ def cli():
 class OdbReader(object):
     def __init__(self, lef_in, def_in):
         self.db = odb.dbDatabase.create()
+
         self.lef = odb.read_lef(self.db, lef_in)
         self.sites = self.lef.getSites()
+
         self.df = odb.read_def(self.db, def_in)
         self.block = self.db.getChip().getBlock()
         self.rows = self.block.getRows()
         self.dbunits = self.block.getDefUnits()
+        self.nets = self.block.getNets()
+
+
+# OdbReader("designs/def_test/runs/RUN_2022.01.30_12.32.26/tmp/merged.lef", "designs/def_test/runs/RUN_2022.01.30_12.32.26/results/final/def/def_test.def")
```

Comment:
No commented out giant test code blocks please.

Reply:
The first line for ODB Reader should be removed.

Second line with openroad, I think I will leave, as it suggests a way to run this script properly, without launching the full flow.

Reply:
Add it to the docstring. Not in a comment.